src/button/button-group.scss

```diff
@@ -7,77 +7,55 @@
   align-items: center;
   isolation: isolate;
 
-  $childSelector: 'input, button'; // target nested inputs and buttons
+  $childZindexSelector: '.iui-input-container, .iui-button, .iui-input:where(:not(.iui-input-container .iui-input))';
+  $childBorderSelector: '.iui-input, .iui-button:not(.iui-borderless)';
+  $disabledSelector: '&:disabled, &.iui-disabled, &[aria-disabled="true"]';
 
   > * {
     display: flex;
 
-    #{$childSelector} {
+    #{$childZindexSelector} {
       position: relative;
 
       &:hover {
         z-index: 1;
       }
 
-      &:focus {
+      &:focus,
+      &:focus-within {
         z-index: 2;
-
-        &:where(:not(:focus-visible):not(:hover)) {
-          z-index: revert;
-        }
       }
 
-      &:disabled {
+      #{$disabledSelector} {
         z-index: -1;
       }
 
+      // Adds stripe above active button
+      --_active-stripe-inset: #{$iui-xxs} #{$iui-xxs} calc(100% - #{$iui-xs});
```

Comment:
What is this naming? Any link to this convention? No need for `iui-button-*0?`

Reply:
It's not an external variable that the user should be using, so I prefer this format. The `_` also indicates that it's private

Reply:
I dont like this. Why we cannot name everything in our library with the prefix `iui` ? Why initiate some new standards, which we have never talked about.

Reply:
`iui-` to me sounds like it's a public variable. What benefit does it provide?

Reply:
added prefix